src/graphql/account.rs

```diff
@@ -1434,4 +1473,80 @@ mod tests {
 
         assert!(res.is_err());
     }
+
+    #[tokio::test]
+    async fn update_language() {
+        let schema = TestSchema::new().await;
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                    insertAccount(
+                        username: "username",
+                        password: "password",
+                        role: "SECURITY_ADMINISTRATOR",
+                        name: "John Doe",
+                        department: "Security",
+                        language: "en-US"
+                    )
+                }"#,
+            )
+            .await;
+
+        assert_eq!(res.data.to_string(), r#"{insertAccount: "username"}"#);
+
+        let res = schema
+            .execute(
+                r#"
+                query {
+                     account(username: "username") {
+                        username
+                        role
+                        name
+                        department
+                        language
+                    }
+                }"#,
+            )
+            .await;
+
+        assert_eq!(
+            res.data.to_string(),
+            r#"{account: {username: "username", role: SECURITY_ADMINISTRATOR, name: "John Doe", department: "Security", language: "en-US"}}"#
+        );
+
+        let res = schema
+            .execute(
+                r#"
+                mutation {
+                    updateLanguage(
+                        username: "username",
+                        language: {
+                            old: "en-US",
+                            new: "ko-KR"
+                        }
+                    )
+                }"#,
+            )
+            .await;
+
+        assert_eq!(res.data.to_string(), r#"{updateLanguage: "username"}"#);
+
+        let res = schema
+            .execute(
+                r#"
+                query {
+                     account(username: "username") {
+                        username
+                        language
```

Comment:
```suggestion
                        username
                        role
                        name
                        department
                        language
```

타이핑을 아주 조금만 더 추가하면 손 쉽게 더 나은 테스트가 될 것 같아서 이렇게 한번 해보실 수 있을까요?  그 이유는 `updateLanguage` API가 호출되었을 때, language만 영향을 받고 다른 것은 그대로 유지된다는 것도 중요한 포인트라고 생각하고 있기 때문입니다.

Reply:
> updateLanguage API가 호출되었을 때, language만 영향을 받고 다른 것은 그대로 유지된다는 것도 중요한 포인트

이에 따라 테스트 코드를 수정해보았습니다.